src/api/Item.js

```diff
@@ -286,29 +286,35 @@ class Item extends Base {
     };
 
     /**
-     * Validate an item update request
+     * Validate request to update an item shared link
      *
      * @param {string|void} itemID - ID of item to share
      * @param {BoxItemPermission|void} itemPermissions - Permissions for item
-     * @param {boolean|void} canSkipSetShareAccess - skip the check for can_set_share_access when creating a new shared link
+     * @param {boolean|void} canSkipSetShareAccessPermission - skip `can_set_share_access` permission check
      * @throws {Error}
      * @return {void}
      */
-    validateRequest(itemID: ?string, itemPermissions: ?BoxItemPermission, canSkipSetShareAccess: boolean = false) {
+    validateSharedLinkRequest(
```

Comment:
this function is only intended to be used for sharing requests (`share()` and `updateSharedLink()`) not any type of request